django_graph_api/views.py

```diff
@@ -0,0 +1,36 @@
+import json
+
+from django.http import JsonResponse
+from django.template.response import TemplateResponse
+from django.views.generic import View
+
+
+class GraphQLView(View):
+    graphiql_version = '0.11.3'
+    graphql_url = '/graphql'
+    template_name = 'django_graph_api/graphiql.html'
+    schema = None
+
+    def get(self, request, *args, **kwargs):
+        return TemplateResponse(
+            request=self.request,
+            template=[self.template_name],
+            context={
+                'graphiql_version': self.graphiql_version,
+                'graphql_url': '/graphql',
```

Comment:
This really should come from the URL configuration / be consistent with `GraphQLView.graphql_url`.